beacon-chain/state/testtmpl/getters_validator.go

```diff
@@ -0,0 +1,22 @@
+package testtmpl
+
+import (
+	"testing"
+
+	"github.com/prysmaticlabs/prysm/beacon-chain/state"
+	"github.com/prysmaticlabs/prysm/testing/assert"
+	"github.com/prysmaticlabs/prysm/testing/require"
+)
+
+type testTmplBeaconState_ValidatorAtIndexReadOnly_HandlesNilSlice_factory func() (state.BeaconState, error)
+
+func TestTmplBeaconState_ValidatorAtIndexReadOnly_HandlesNilSlice(
```

Comment:
```suggestion
func TestBeaconState_ValidatorAtIndexReadOnly_HandlesNilSlice(
```

Reply:
@terencechain there would be no way of disguising a test from the test template by the function name